_data/data.yml

```diff
@@ -2579,50 +2578,58 @@ and the Debian popularity contest. Our tests found a total of 63 bugs in 31 appl
   that most of these bugs would likely be contained in our
   sandbox."
 
-    - &cappos_seattle_sigcse_2009
-      anchor: cappos_seattle_sigcse_2009
-      title: "Seattle: A Platform for Educational Cloud Computing"
-      authors:
-        - name: "J. Cappos, I. Beschastnikh, A. Krishnamurthy, and T. Anderson"
-      project: *seattle
-      booktitle: "Proceedings of the 40th ACM Technical Symposium on Computer Science Education (SIGCSE '09)"
-      series:
-      year:
-      isbn:
-      location:
-      pages: "111--115"
-      numpages: "5"
-      acmid:
-      publisher: "ACM"
-      address:
-      link: "/papers/cappos_seattle_sigcse_2009.pdf"
-      keywords: "cloud computing, peer-to-peer computing, cluster computing, distributed computing"
-      abstract: "Cloud computing is rapidly increasing in popularity. Companies
-  such as RedHat, Microsoft, Amazon, Goog
-  le, and
-  IBM are increasingly funding cloud computing infrastructure
-  and research, making it important for students to gain
-  the necessary skills to work with cloud-based resources. This
-  paper presents a free, educational research platform called
-  Seattle that is community-driven, a common denominator
-  for diverse platform types, and is broadly deployed.
-  Seattle is community-driven - universities donate available
-  compute resources on multi-user machines to the platform.
-  These donations can come from systems with a wide
-  variety of systems operating and architectures, removing the
-  need for a dedicated infrastructure.
-  Seattle is also surprisingly flexible and supports a variety
-  of pedagogical uses because as a platform it represents a
-  common denominator for cloud computing, grid computing,
-  peer-to-peer networking, distributed systems, and networking.
-  Seattle programs are portable. Students' code can run
-  across different operating systems and architectures without
-  change, while the Seattle programming language is expressive
-  enough for experimentation at a fine-grained level. Our
-  current deployment of Seattle consists of about one thousand
-  computers that are distributed around the world. We
-  invite the computer science education community to employ
-  Seattle in their courses."
+  - &cappos_seattle_sigcse_2009
```

Comment:
This seems to be the problem.  What was changed here?  It looks like indentation to me.

Reply:
This may have gotten out of line when I was adding in the Uptane standard. I'll try to adjust the indentation tomorrow. The thing is, when I check the test results, it points to line numbers around the TUF description. That's where I've been trying to look for issues, but I've fixed the only things that may have been problematic there. Anyway, thanks for looking. I think from here on out I need to do really small commits, so if I make a mistake, its easier to find it.